django/core/files/base.py

```diff
@@ -105,11 +105,11 @@ def __enter__(self):
     def __exit__(self, exc_type, exc_value, tb):
         self.close()
 
-    def open(self, mode=None):
+    def open(self, mode=None, *args, **kwargs):
```

Comment:
This change should be [documented](https://docs.djangoproject.com/en/4.2/ref/files/file/#django.core.files.File.open) with a `versionchanged` annotation and release note (`5.0.txt`).